### ISSUE TITLE
[Accessibility] Python native repl: Cannot move focus to executed output cell

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#233238]
CC @meganrogge

ctrl+UpArrow in Python native repl goes to the list item where users can find the input history just like Jupyter Notebook input cells. Currently, there is no way to move focus to each of the corresponding output cells. It is ctrl+DownArrow in Jupyter Notebook, but ctrl+DownArrow moves to the repl input area.

**Comment by meganrogge** [comment]
I believe @amunger is adding commands to fix this now?

**Comment by amunger** [comment]
yes, I made it ctrl+shift+down perform the same action in insiders because of that conflict, though the a11y help menu still says ctrl+down so I'll see if I can update that.

**Comment by amunger** [comment]
you can also open the a11y view for output just by focusing on the cell container now in insiders, you don't need to focus the output first

**Comment by meganrogge** [comment]
want to keep this open for anything @amunger or shall we close it?

**Comment by amunger** [comment]
Just for trying to update the a11y help
